qa/cephfs/overrides/ignorelist_health.yaml

```diff
@@ -2,13 +2,15 @@ overrides:
   ceph:
     log-ignorelist:
       - FS_DEGRADED
+      - fs.*is degraded
       - FS_INLINE_DATA_DEPRECATED
```

Comment:
For this maybe we also need to add the following string:

`fs cephfs has deprecated feature inline_data enabled`

Reply:
Okay, have we seen failure due to this? I didn't find it in QA runs I was reviewing...